tests/integration/cli/test_cache.py

```diff
@@ -5,6 +5,6 @@
 def test_cache_init_purge(ape_cli, runner):
     cmd = ("cache", "init", "--network", "ethereum:sepolia")
     result = runner.invoke(ape_cli, cmd)
-    assert result.output == "SUCCESS: Caching database initialized for ethereum:sepolia.\n"
+    assert result.output == "SUCCESS : Caching database initialized for ethereum:sepolia.\n"
     result = runner.invoke(ape_cli, ("cache", "purge", "--network", "ethereum:sepolia"))
-    assert result.output == "SUCCESS: Caching database purged for ethereum:sepolia.\n"
+    assert result.output == "SUCCESS : Caching database purged for ethereum:sepolia.\n"
```

Comment:
i think the padding should occur after the `:`

Reply:
It's kind of hard, due to how the plugins display

Reply:
Wonder if we should just get rid of the :?

Reply:
probably. tbh alignment here seems strange to me. only the log levels are aligned and no other parts of the logs.